web/pano_editor.css

```
@@ -1234,26 +1285,26 @@
+.pano-picker-item-icon {
+  width: 14px;
+  height: 14px;
+  flex: 0 0 14px;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+}
+.pano-picker-item-icon svg {
+  width: 14px;
+  height: 14px;
+  stroke: currentColor;
```

Stylelint (17.6.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)